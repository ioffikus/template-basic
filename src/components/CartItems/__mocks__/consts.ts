const pendingAPI = 1000;

export { pendingAPI };
