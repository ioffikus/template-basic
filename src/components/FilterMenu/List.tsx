import React from 'react';
import { Radio } from 'antd';
import _ from 'lodash';

interface IProps {
  ui: Alicanto.Models.UI;
  value: number[];
  fieldSlug: string;
  filterType: string;
  valueType: string;
  categoriesResults: Alicanto.Models.Category[];
  onChange: (value: number[], fieldSlug: string, valueType: string, filterType: string) => void;
}

interface IState {
  value: number;
}

class List extends React.PureComponent<IProps, IState> {
  readonly defauldValue = 'all';

  handleChange = (event: React.SyntheticEvent<any>) => {
    const value: 'all' | number = _.get(event.target, 'value', 'all');
    const nValue = value !== 'all' ? [value] : null;
    this.props.onChange(nValue, this.props.fieldSlug, this.props.valueType, this.props.filterType);
  };

  render() {
    const commonUI = this.props.ui.common;
    let value: string | number = this.defauldValue;
    if (!_.isNil(this.props.value)) {
      value = this.props.value[0];
    }

    return (
      <div className="root">
        <style jsx>{`
          .root :global(.filters) {
            max-width: 100%;
          }
          .root :global(.filters .ant-radio-button-wrapper) {
            display: flex;
            align-items: center;
            line-height: 1.5;
            height: auto;
            min-height: 20px;
            white-space: normal;
            padding-top: 4px;
            padding-bottom: 4px;
          }
          .root :global(.filters .ant-radio-button-wrapper + .ant-radio-button-wrapper) {
            margin-top: 10px;
          }
        `}</style>
        <Radio.Group onChange={this.handleChange} value={value} className="filters">
          <Radio.Button value={this.defauldValue}>
            {commonUI.filters.categories.items.allCategories.i18n.defaultText}
          </Radio.Button>
          {this.props.categoriesResults.map((item: Alicanto.Models.Category) => (
            <Radio.Button key={item.id} value={item.id}>
              {item.name}
            </Radio.Button>
          ))}
        </Radio.Group>
      </div>
    );
  }
}

export default List;
