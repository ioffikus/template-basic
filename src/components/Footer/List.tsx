import React from 'react';
import { Row, Col } from 'antd';
import Item from './Item';

interface IProps {
  ui: Alicanto.Models.UI;
}

interface IState {}

class List extends React.Component<IProps, IState> {
  render() {
    const items = this.props.ui.common.footerLinks.items;
    return (
      <div className="root app-container">
        <style jsx>{`
          .root :global(.item) {
            flex: 1;
          }
        `}</style>
        <Row type="flex">
          {Object.keys(items).map(key => {
            const item = items[key];
            return (
              <Col className="item" key={key}>
                <Item links={item.links} title={item.title.i18n.defaultText} />
              </Col>
            );
          })}
        </Row>
      </div>
    );
  }
}

export default List;
