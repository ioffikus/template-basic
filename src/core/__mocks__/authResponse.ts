const authResponse = {
  data: { username: 'user', password: 'password' },
  headers: { 'x-shop-domain': 'template-basic.agg.loc' },
  method: 'POST',
  url: '/login',
};

export default authResponse;
