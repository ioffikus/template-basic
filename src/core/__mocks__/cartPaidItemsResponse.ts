const cartPaidItemsResponse = {
  count: 1,
  next: null as string,
  previous: null as string,
  results: [
    {
      id: 27,
      item: '74212e0f54674aed8964',
      uid: '74212e0f54674aed8964',
      modified: '2017-10-12T09:43:31.442202Z',
      count: 1,
      in_stock: true,
      is_deleted: false,
      schema_fields: {
        name: { type: 'string', label: 'Name', value: 'bla eng bla' },
        short_description: { type: 'text', label: 'Short description', value: 'english bla bla' },
        price: { type: 'decimal', label: 'Price', value: 0 },
        currency: { type: 'string', label: 'Currency', value: '€' },
        picture_url: {
          type: 'string',
          label: 'Picture',
          value:
            'https://d29vc6x3bmmp7p.cloudfront.net/item_pictures/2017-06-05T072530.24449100008df588254089446ab7a52213a70be4c2.jpg',
        },
      },
      aggregion_catalog_item_url: 'agg://collection/58c6ac00a4c8da4900e9024f/bundle',
    },
    {
      id: 108,
      item: 'a9e71d42bab344c9ba1b',
      uid: 'a9e71d42bab344c9ba1b',
      modified: '2017-11-01T10:17:58.183903Z',
      count: 1,
      in_stock: true,
      is_deleted: false,
      schema_fields: {
        name: { type: 'string', value: 'Eng The Last Wish: Introducing (The Witcher)', label: 'Name' },
        short_description: { type: 'text', value: 'Eng description', label: 'Short description' },
        picture_url: {
          type: 'string',
          value:
            'https://d29vc6x3bmmp7p.cloudfront.net/item_pictures/2017-06-05T072535.7689600000119452d41e8941ee85510e6517050b31.jpg',
          label: 'Picture',
        },
        currency: { type: 'string', value: '€', label: 'Currency' },
        price: { type: 'decimal', value: 0, label: 'Price' },
      },
      aggregion_catalog_item_url: 'agg://collection/58d41d54a3fab345003aee61/bundle',
    },
  ],
};

export default cartPaidItemsResponse;
