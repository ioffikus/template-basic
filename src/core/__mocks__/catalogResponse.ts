const results = [
  {
    uid: '81a0709e8a9c4869b875',
    schema_fields: {
      picture_url: {
        value:
          'https://d29vc6x3bmmp7p.cloudfront.net/item_pictures/2017-09-14T194901.55022700002b7db5e1dc434111ac8de14df22fd54c.jpg',
        label: 'Картинка',
        type: 'string',
      },
      currency: { value: '€', label: 'Валюта', type: 'string' },
      name: { value: 'A Clash of Kings (A Song of Ice and Fire, Book 2)', label: 'Имя', type: 'string' },
      price: { value: 0, label: 'Цена', type: 'decimal' },
      short_description: { value: 'A comet the color of', label: 'Краткое описание', type: 'text' },
    },
    was_acquired_before: false,
    is_bundle: false,
  },
  {
    uid: '6ab8164eea2b46d6a848',
    schema_fields: {
      picture_url: {
        value:
          'https://d29vc6x3bmmp7p.cloudfront.net/item_pictures/2017-09-14T194903.2725290000aafcd25f08e046178e86ce33ba31db52.jpg',
        label: 'Картинка',
        type: 'string',
      },
      currency: { value: '€', label: 'Валюта', type: 'string' },
      name: { value: 'A Game of Thrones (A Song of Ice and Fire, Book 1)', label: 'Имя', type: 'string' },
      price: { value: 0.01, label: 'Цена', type: 'decimal' },
      short_description: { value: 'A Game of Thrones is', label: 'Краткое описание', type: 'text' },
    },
    was_acquired_before: false,
    is_bundle: false,
  },
  {
    uid: '3b6103b22d594ad7a354',
    schema_fields: {
      picture_url: {
        value:
          'https://d29vc6x3bmmp7p.cloudfront.net/item_pictures/2017-09-14T194903.72818400006075338fcf2f40848210d00b419255c0.jpg',
        label: 'Картинка',
        type: 'string',
      },
      currency: { value: '€', label: 'Валюта', type: 'string' },
      name: {
        value: 'The Two Towers: Being the Second Part of The Lord of the Rings',
        label: 'Имя',
        type: 'string',
      },
      price: { value: 0, label: 'Цена', type: 'decimal' },
      short_description: { value: 'The second volume in', label: 'Краткое описание', type: 'text' },
    },
    was_acquired_before: false,
    is_bundle: false,
  },
  {
    uid: 'd6c3bc0ef9ee4e67a7d3',
    schema_fields: {
      picture_url: {
        value:
          'https://d29vc6x3bmmp7p.cloudfront.net/item_pictures/2017-09-14T194904.3079870000e8725d8f98b94fa5a4a12efa71a6ed42.jpg',
        label: 'Картинка',
        type: 'string',
      },
      currency: { value: '€', label: 'Валюта', type: 'string' },
      name: {
        value: 'The Fellowship of the Ring: Being the First Part of The Lord of the Rings',
        label: 'Имя',
        type: 'string',
      },
      price: { value: 0.01, label: 'Цена', type: 'decimal' },
      short_description: { value: 'The first volume in ', label: 'Краткое описание', type: 'text' },
    },
    was_acquired_before: false,
    is_bundle: false,
  },
  {
    uid: 'd12836ad085f477fbdd7',
    schema_fields: {
      picture_url: {
        value:
          'https://d29vc6x3bmmp7p.cloudfront.net/item_pictures/2017-09-14T194905.078196000088c60a6aa31f4d91ad4e4c9f2afce088.jpg',
        label: 'Картинка',
        type: 'string',
      },
      currency: { value: '€', label: 'Валюта', type: 'string' },
      name: { value: 'Test new price', label: 'Имя', type: 'string' },
      price: { value: 0.02, label: 'Цена', type: 'decimal' },
      short_description: { value: '', label: 'Краткое описание', type: 'text' },
    },
    was_acquired_before: false,
    is_bundle: false,
  },
  {
    uid: '6a744b3c24a040f6a6fc',
    schema_fields: {
      picture_url: {
        value:
          'https://d29vc6x3bmmp7p.cloudfront.net/item_pictures/2017-09-14T194905.5909400000d1d92eef29874f7a83a45c50d14449be.JPG',
        label: 'Картинка',
        type: 'string',
      },
      currency: { value: '€', label: 'Валюта', type: 'string' },
      name: { value: 'Test custom fields1', label: 'Имя', type: 'string' },
      price: { value: 0.01, label: 'Цена', type: 'decimal' },
      short_description: { value: 'test description1', label: 'Краткое описание', type: 'text' },
    },
    was_acquired_before: false,
    is_bundle: false,
  },
  {
    uid: 'a8a4fd6a58cf49eb81d8',
    schema_fields: {
      picture_url: {
        value:
          'https://d29vc6x3bmmp7p.cloudfront.net/item_pictures/2017-09-14T194906.6056180000ca6d7162b7934a04bbdfcedf0189a61c.jpg',
        label: 'Картинка',
        type: 'string',
      },
      currency: { value: '€', label: 'Валюта', type: 'string' },
      name: { value: 'Test  images11', label: 'Имя', type: 'string' },
      price: { value: 0.01, label: 'Цена', type: 'decimal' },
      short_description: { value: '111', label: 'Краткое описание', type: 'text' },
    },
    was_acquired_before: false,
    is_bundle: false,
  },
  {
    uid: 'fbfa32c1f4144ac49735',
    schema_fields: {
      picture_url: {
        value:
          'https://d29vc6x3bmmp7p.cloudfront.net/item_pictures/2017-09-14T194909.4265800000f747a4b8288d48c3ba460354205e7d7a.JPG',
        label: 'Картинка',
        type: 'string',
      },
      currency: { value: '€', label: 'Валюта', type: 'string' },
      name: { value: 'test additional image', label: 'Имя', type: 'string' },
      price: { value: 0, label: 'Цена', type: 'decimal' },
      short_description: { value: '', label: 'Краткое описание', type: 'text' },
    },
    was_acquired_before: false,
    is_bundle: false,
  },
  {
    uid: '44b4f474594e4a5d873e',
    schema_fields: {
      picture_url: {
        value:
          'https://d29vc6x3bmmp7p.cloudfront.net/item_pictures/2017-09-14T194910.9746300000499aa53968e1460ba4156e77e598e591.jpg',
        label: 'Картинка',
        type: 'string',
      },
      currency: { value: '$', label: 'Валюта', type: 'string' },
      name: { value: 'Sword of Destiny (The Witcher)', label: 'Имя', type: 'string' },
      price: { value: 0.01, label: 'Цена', type: 'decimal' },
      short_description: { value: 'The New York Times b', label: 'Краткое описание', type: 'text' },
    },
    was_acquired_before: false,
    is_bundle: false,
  },
  {
    uid: '1810236f7f1d4f09ac32',
    schema_fields: {
      picture_url: {
        value:
          'https://d29vc6x3bmmp7p.cloudfront.net/item_pictures/2017-09-14T194911.45059300001ea1a0a392c04b8b8797567dbdc07b60.jpg',
        label: 'Картинка',
        type: 'string',
      },
      currency: { value: '€', label: 'Валюта', type: 'string' },
      name: { value: 'Blood of Elves (The Witcher)', label: 'Имя', type: 'string' },
      price: { value: 0, label: 'Цена', type: 'decimal' },
      short_description: { value: 'The New York Times b', label: 'Краткое описание', type: 'text' },
    },
    was_acquired_before: false,
    is_bundle: false,
  },
  {
    uid: 'db0096c990344b35bcf5',
    schema_fields: {
      picture_url: {
        value:
          'https://d29vc6x3bmmp7p.cloudfront.net/item_pictures/2017-09-14T194912.0003190000c3ec02ab2ea047f592ee1b062bf2dd1e.jpg',
        label: 'Картинка',
        type: 'string',
      },
      currency: { value: '€', label: 'Валюта', type: 'string' },
      name: { value: 'The Last Wish: Introducing (The Witcher)', label: 'Имя', type: 'string' },
      price: { value: 0.02, label: 'Цена', type: 'decimal' },
      short_description: { value: 'The New York Times b', label: 'Краткое описание', type: 'text' },
    },
    was_acquired_before: false,
    is_bundle: false,
  },
  {
    uid: 'f3dfcf3e174a40b6a191',
    schema_fields: {
      picture_url: {
        value:
          'https://d29vc6x3bmmp7p.cloudfront.net/item_pictures/2017-09-14T194912.5094260000445dc4e20391450cb1f9ff609f187bf3.jpg',
        label: 'Картинка',
        type: 'string',
      },
      currency: { value: '€', label: 'Валюта', type: 'string' },
      name: { value: 'The Time of Contempt (The Witcher)', label: 'Имя', type: 'string' },
      price: { value: 0, label: 'Цена', type: 'decimal' },
      short_description: { value: 'The New York Times b', label: 'Краткое описание', type: 'text' },
    },
    was_acquired_before: false,
    is_bundle: false,
  },
  {
    uid: 'b1f91e8db90e4d6d9401',
    schema_fields: {
      picture_url: {
        value:
          'https://d29vc6x3bmmp7p.cloudfront.net/item_pictures/2017-09-14T194913.099207000071aa46bd29d34798977c01592f69062d.jpg',
        label: 'Картинка',
        type: 'string',
      },
      currency: { value: '€', label: 'Валюта', type: 'string' },
      name: { value: 'Baptism of Fire (The Witcher)', label: 'Имя', type: 'string' },
      price: { value: 0.01, label: 'Цена', type: 'decimal' },
      short_description: { value: 'The New York Times b', label: 'Краткое описание', type: 'text' },
    },
    was_acquired_before: false,
    is_bundle: false,
  },
  {
    uid: '4fac21dfec49475db64d',
    schema_fields: {
      picture_url: {
        value:
          'https://d29vc6x3bmmp7p.cloudfront.net/item_pictures/2017-09-14T194915.93251300002ecebdf6dd0946ceb665a6a4f7658b56.jpg',
        label: 'Картинка',
        type: 'string',
      },
      currency: { value: '€', label: 'Валюта', type: 'string' },
      name: { value: 'The Tower of Swallows (The Witcher)', label: 'Имя', type: 'string' },
      price: { value: 0.01, label: 'Цена', type: 'decimal' },
      short_description: { value: 'The Witcher returns ', label: 'Краткое описание', type: 'text' },
    },
    was_acquired_before: false,
    is_bundle: false,
  },
  {
    uid: 'fc4ffd07dc294a039186',
    schema_fields: {
      picture_url: {
        value:
          'https://d29vc6x3bmmp7p.cloudfront.net/item_pictures/2017-09-14T194916.9685320000f08918cf317b43c2bbe8911f4d7695a6.jpg',
        label: 'Картинка',
        type: 'string',
      },
      currency: { value: '€', label: 'Валюта', type: 'string' },
      name: { value: 'ведьмак', label: 'Имя', type: 'string' },
      price: { value: 0.01, label: 'Цена', type: 'decimal' },
      short_description: { value: 'ололо', label: 'Краткое описание', type: 'text' },
    },
    was_acquired_before: false,
    is_bundle: false,
  },
  {
    uid: 'f5f4b0eb3dc248939ae8',
    schema_fields: {
      picture_url: {
        value:
          'https://d29vc6x3bmmp7p.cloudfront.net/item_pictures/2017-09-14T194917.5601610000a4f086c39efc48c0954dc19375e9c602.jpg',
        label: 'Картинка',
        type: 'string',
      },
      currency: { value: '€', label: 'Валюта', type: 'string' },
      name: { value: 'Test new', label: 'Имя', type: 'string' },
      price: { value: 0, label: 'Цена', type: 'decimal' },
      short_description: { value: 'Test description', label: 'Краткое описание', type: 'text' },
    },
    was_acquired_before: false,
    is_bundle: true,
  },
  {
    uid: '57f2591404c941eab006',
    schema_fields: {
      picture_url: {
        value:
          'https://d29vc6x3bmmp7p.cloudfront.net/item_pictures/2017-09-14T194907.06655900001e9011a11be14b0eb551d591d6ee8f1a.jpg',
        label: 'Картинка',
        type: 'string',
      },
      currency: { value: '€', label: 'Валюта', type: 'string' },
      name: { value: 'Test bundle', label: 'Имя', type: 'string' },
      price: { value: 0, label: 'Цена', type: 'decimal' },
      short_description: { value: 'Test bundle descript', label: 'Краткое описание', type: 'text' },
    },
    was_acquired_before: false,
    is_bundle: true,
  },
  {
    uid: '8119f05cb5e24211b388',
    schema_fields: {
      picture_url: {
        value:
          'https://d29vc6x3bmmp7p.cloudfront.net/item_pictures/2017-09-14T194919.7744850000fa00eb4588e7409f9ee8f6f5687ef14e.JPG',
        label: 'Картинка',
        type: 'string',
      },
      currency: { value: '€', label: 'Валюта', type: 'string' },
      name: { value: 'Test outer', label: 'Имя', type: 'string' },
      price: { value: 0, label: 'Цена', type: 'decimal' },
      short_description: { value: '', label: 'Краткое описание', type: 'text' },
    },
    was_acquired_before: false,
    is_bundle: true,
  },
  {
    uid: '598c39a19090483bb33b',
    schema_fields: {
      picture_url: {
        value:
          'https://d29vc6x3bmmp7p.cloudfront.net/item_pictures/2017-09-14T194921.59418800003856d57bca22484daf8a1fb6fcc8c597.jpg',
        label: 'Картинка',
        type: 'string',
      },
      currency: { value: '€', label: 'Валюта', type: 'string' },
      name: { value: 'test new QA', label: 'Имя', type: 'string' },
      price: { value: 0, label: 'Цена', type: 'decimal' },
      short_description: { value: 'dafsdfasdfadfs', label: 'Краткое описание', type: 'text' },
    },
    was_acquired_before: false,
    is_bundle: false,
  },
];

const count = 19;

const catalogResponse: Alicanto.API.GET_CATALOG_RESPONSE = {
  results,
  count,
  next: null as string,
  previous: null as string,
};

export default catalogResponse;
