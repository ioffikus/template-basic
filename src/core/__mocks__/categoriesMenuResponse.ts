const categoriesMenuResponse = {
  slug: 'categories_menu',
  type: 'categories',
  content: [
    {
      id: 2,
      uid: '3e6736da4e2c46c48c0c',
      name: 'Science fiction',
      subcategories: [
        {
          id: 3,
          uid: 'fdd18449106641f2a236',
          name: 'Alien invasion',
        },
        {
          id: 5,
          uid: '4c7ed4040e0a4df3986b',
          name: 'Cyberpunk derivatives',
          subcategories: [
            {
              id: 1,
              uid: '5e6736da4e2c46c48c0c',
              name: 'Level 3',
            },
          ],
        },
      ],
    },
    {
      id: 6,
      uid: 'cee8c7fc0ea74215a3bb',
      name: 'Cyberpunk',
      subcategories: [] as {},
    },
  ],
};

export default categoriesMenuResponse;
