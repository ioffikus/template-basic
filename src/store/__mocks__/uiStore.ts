import { IUIState } from 'src/ducks/ui';
import fs from 'fs';
import yaml from 'js-yaml';

const globalConfig = yaml.safeLoad(fs.readFileSync('src/configs/global.yml', 'utf8'));
export const ui = globalConfig.ui;

const state: IUIState = {
  ...ui,
  isRequestError: false,
  isRequesting: false,
};

export default state;
