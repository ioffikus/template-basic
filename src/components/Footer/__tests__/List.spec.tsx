import React from 'react';
import { shallow } from 'enzyme';
import { ui } from 'src/store/__mocks__/uiStore';
import List from '../List';

describe(null, () => {
  it('render() correctly', () => {
    const wrapper = shallow(<List ui={ui} />);
    expect(wrapper).toMatchSnapshot();
  });
});
