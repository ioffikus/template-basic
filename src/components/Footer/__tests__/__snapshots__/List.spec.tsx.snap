// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render() correctly 1`] = `
<div
  className="jsx-3150404229 root app-container"
>
  <JSXStyle
    css=".root.jsx-3150404229 .item{-webkit-flex:1;-ms-flex:1;flex:1;}"
    styleId="3150404229"
  />
  <Row
    gutter={0}
    type="flex"
  >
    <Col
      className="item"
      key="about"
    >
      <Item
        links={
          Object {
            "/link1": Object {
              "text": Object {
                "i18n": Object {
                  "defaultText": "Our store",
                  "description": "Текст ссылки на страницу Our store",
                },
              },
              "title": Object {
                "i18n": Object {
                  "defaultText": "Our store",
                  "description": "Подсказка ссылки на страницу Our store",
                },
              },
            },
            "/link2": Object {
              "text": Object {
                "i18n": Object {
                  "defaultText": "About company",
                  "description": "Текст ссылки на страницу About company",
                },
              },
              "title": Object {
                "i18n": Object {
                  "defaultText": "About company",
                  "description": "Подсказка ссылки на страницу About company",
                },
              },
            },
            "/link3": Object {
              "text": Object {
                "i18n": Object {
                  "defaultText": "Terms & conditions",
                  "description": "Текст ссылки на страницу Terms & conditions",
                },
              },
              "title": Object {
                "i18n": Object {
                  "defaultText": "Terms & conditions",
                  "description": "Подсказка ссылки на страницу Terms & conditions",
                },
              },
            },
          }
        }
        title="About"
      />
    </Col>
    <Col
      className="item"
      key="support"
    >
      <Item
        links={
          Object {
            "/link1": Object {
              "text": Object {
                "i18n": Object {
                  "defaultText": "FAQ",
                  "description": "Текст ссылки на страницу FAQ",
                },
              },
              "title": Object {
                "i18n": Object {
                  "defaultText": "FAQ",
                  "description": "Подсказка ссылки на страницу FAQ",
                },
              },
            },
            "/link2": Object {
              "text": Object {
                "i18n": Object {
                  "defaultText": "Support team",
                  "description": "Текст ссылки на страницу Support team",
                },
              },
              "title": Object {
                "i18n": Object {
                  "defaultText": "Support team",
                  "description": "Подсказка ссылки на страницу Support team",
                },
              },
            },
            "/link3": Object {
              "text": Object {
                "i18n": Object {
                  "defaultText": "Feedback",
                  "description": "Текст ссылки на страницу Feedback",
                },
              },
              "title": Object {
                "i18n": Object {
                  "defaultText": "Feedback",
                  "description": "Подсказка ссылки на страницу Feedback",
                },
              },
            },
          }
        }
        title="Support"
      />
    </Col>
    <Col
      className="item"
      key="payment"
    >
      <Item
        links={
          Object {
            "/link1": Object {
              "text": Object {
                "i18n": Object {
                  "defaultText": "Payment methods",
                  "description": "Текст ссылки на страницу Payment methods",
                },
              },
              "title": Object {
                "i18n": Object {
                  "defaultText": "Payment methods",
                  "description": "Подсказка ссылки на страницу Payment methods",
                },
              },
            },
            "/link2": Object {
              "text": Object {
                "i18n": Object {
                  "defaultText": "Free delivery",
                  "description": "Текст ссылки на страницу Free delivery",
                },
              },
              "title": Object {
                "i18n": Object {
                  "defaultText": "Free delivery",
                  "description": "Подсказка ссылки на страницу Free delivery",
                },
              },
            },
          }
        }
        title="Payment and delivery"
      />
    </Col>
  </Row>
</div>
`;
